go.mod

```diff
@@ -15,11 +15,15 @@
 module gocloud.dev
 
 require (
-	cloud.google.com/go v0.35.0
+	cloud.google.com/go v0.34.0
```

Comment:
Looks like you need to revert your `go.mod` and `go.sum` changes.

Reply:
Ack, I messed up the Git command to do so. Sigh...